avail/src/main/kotlin/avail/anvil/AvailWorkbench.kt

```diff
@@ -1155,6 +1155,7 @@ class AvailWorkbench internal constructor(
 				val beforeRemove = System.nanoTime()
 				document.remove(
 					statusSize, min(amountToRemove, length - statusSize))
+				transcript.undoManager.discardAllEdits()
```

Comment:
Was this the only `UndoManager` problem, do you think? I know that we were having problems, but was this the big culprit?